assets/style.css

```diff
@@ -461,6 +461,45 @@ body {
         text-decoration: none;
 }
 
+.hide {
+        display: none;
+}
+
+.popup {
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        position: fixed;
+        z-index: 1000000;
+        background-color: #fdfdfd;
+}
+
+.close_icon {
+        text-align: right;
+}
+
+.popup-content {
+        max-width: 85%;
+        margin: 20px auto;
+}
+
+.popup_menu_option {
+        list-style-type: none;
+        margin-bottom: 20px;
+}
+
+.popup_menu_option a {
+        text-decoration: none;
+        color: #3A4A42;
+        font-size: 25px;
+        font-weight: 600;
+}
```

Comment:
- [ ] It would be nice if you could use the exact font family for the popup links just as it is in the design and as required in the project requirements to follow all design guidelines as possible.

Add this style `font-family: "Inter", sans-serif;`

**See the Template Design in Figma**
![image](https://user-images.githubusercontent.com/50941074/194911262-bc937f16-4074-4f0e-b093-01040d5510b2.png)

**Your Design is having Times New Roman**
![image](https://user-images.githubusercontent.com/50941074/194911453-375d6af1-4255-46ae-bf96-56c8f9895489.png)